ceph/ceph.py

```diff
@@ -607,9 +607,12 @@ def get_osd_metadata(self, osd_id, client=None):
                 return metadata
         return None
 
-    def osd_check(self, client):
+    def osd_check(self, rhbuild, client):
+        cmd = "ceph -s -f json"
```

Comment:
IMO, we shouldnt attempt to change the method signature. That i.e., we could do `def osd_check(self, client, rhbuild=None):`

Reply:
I did noticed osd_check was used only in check_health, but however this makes sense.

Reply:
Added changes.